Node panics when receiving a ConfirmLoop packet from SetupNode.

> Branch: `mainnet-dev`

Logs provided by @ivcosla :
```
[2019-05-01T11:32:06+02:00] INFO [router]: Got new Setup request with type ConfirmLoop
panic: invalid rule

goroutine 143 [running]:
github.com/skycoin/skywire/pkg/routing.Rule.RemotePK(0x7fc6ec6c408b, 0x1d, 0x1d, 0x0, 0x0, 0x0, 0x0, 0x3fe0000000000000)
        /home/ivan/Desktop/skycoin/skywire/pkg/routing/rule.go:74 +0x174
github.com/skycoin/skywire/pkg/router.(*RoutingTableManager).FindAppRule.func1(0x13, 0x7fc6ec6c408b, 0x1d, 0x1d, 0x0)
        /home/ivan/Desktop/skycoin/skywire/pkg/router/routingtablemanager.go:96 +0x93
github.com/skycoin/skywire/pkg/routing.(*boltDBRoutingTable).RangeRules.func1.1(0x7fc6ec6c4087, 0x4, 0x4, 0x7fc6ec6c408b, 0x1d, 0x1d, 0x1d, 0xc000552040)
        /home/ivan/Desktop/skycoin/skywire/pkg/routing/boltdb_routing_table.go:83 +0x64
go.etcd.io/bbolt.(*Bucket).ForEach(0xc000037b80, 0xc0005199f0, 0x7, 0x7)
        /home/ivan/Desktop/skycoin/pkg/mod/go.etcd.io/bbolt@v1.3.2/bucket.go:388 +0xfe
github.com/skycoin/skywire/pkg/routing.(*boltDBRoutingTable).RangeRules.func1(0xc00042c1c0, 0x9f6620, 0xc00042c1c0)
        /home/ivan/Desktop/skycoin/skywire/pkg/routing/boltdb_routing_table.go:82 +0x99
go.etcd.io/bbolt.(*DB).View(0xc0001f2200, 0xc000519a80, 0x0, 0x0)
        /home/ivan/Desktop/skycoin/pkg/mod/go.etcd.io/bbolt@v1.3.2/db.go:719 +0x90
github.com/skycoin/skywire/pkg/routing.(*boltDBRoutingTable).RangeRules(0xc0000105a8, 0xc0000dc1c0, 0x199, 0x0)
        /home/ivan/Desktop/skycoin/skywire/pkg/routing/boltdb_routing_table.go:80 +0x56
github.com/skycoin/skywire/pkg/router.(*RoutingTableManager).FindAppRule(0xc000037a80, 0xdb420fbddbe7a403, 0x866357a30c31d7a, 0xbba79093d212ce35, 0x5b35b37a6f9a483c, 0xd240380200020009, 0x4bc9d86a51e0e5f3, 0x8ec0ac4aaca03b75, 0x11ea1e67ddec32b2, 0xa00759561cb7b, ...)
        /home/ivan/Desktop/skycoin/skywire/pkg/router/routingtablemanager.go:93 +0x101
github.com/skycoin/skywire/pkg/router.(*router).handleSetup.func3(0x51e0e5f3d2403802, 0xaca03b754bc9d86a, 0xddec32b28ec0ac4a, 0x9561cb7b11ea1e67, 0x2000a0075, 0x15, 0xc000556180, 0x31, 0x33, 0xc00047e5e8, ...)
        /home/ivan/Desktop/skycoin/skywire/pkg/router/router.go:277 +0x21d
github.com/skycoin/skywire/pkg/router.(*router).handleSetup(0xc00025c840, 0xa96800, 0xc000037ac0, 0xa8cd20, 0xc0003949a0, 0xc000340f00, 0xa8cd20)
        /home/ivan/Desktop/skycoin/skywire/pkg/router/router.go:358 +0x98c
github.com/skycoin/skywire/pkg/router.(*router).Serve.func2(0xa959c0, 0xc0003949a0, 0xc0003185b0)
        /home/ivan/Desktop/skycoin/skywire/pkg/router/router.go:90 +0x97
created by github.com/skycoin/skywire/pkg/router.(*router).Serve.func3
        /home/ivan/Desktop/skycoin/skywire/pkg/router/router.go:109 +0x1a9


```

--- a/pkg/router/routingtablemanager.go
+++ b/pkg/router/routingtablemanager.go
@@ -90,14 +90,16 @@ func (rtm *RoutingTableManager) DeleteAppRule(lm app.LoopMeta) error {
 
 // FindAppRule finds an APP rule with given LoopMeta.
 func (rtm *RoutingTableManager) FindAppRule(lm app.LoopMeta) (rtID routing.RouteID, rule routing.Rule, ok bool) {
-	_ = rtm.RangeRules(func(id routing.RouteID, r routing.Rule) bool { //nolint:errcheck
+	_ = rtm.RangeRules(func(id routing.RouteID, r routing.Rule) (next bool) { //nolint:errcheck
+		if r.Type() != routing.RuleApp {
+			return true
+		}
 		var (
-			typesMatch  = r.Type() == routing.RuleApp
 			rPKsMatch   = r.RemotePK() == lm.Remote.PubKey
 			rPortsMatch = r.RemotePort() == lm.Remote.Port
 			lPortsMatch = r.LocalPort() == lm.Local.Port
 		)
-		if typesMatch && rPKsMatch && rPortsMatch && lPortsMatch {
+		if rPKsMatch && rPortsMatch && lPortsMatch {
 			rtID, rule, ok = id, r, true
 			return false
 		}

